components/add_new_cv_form.tsx

```diff
@@ -34,44 +31,54 @@ interface Props {
 
 export default function AddNewCvForm({ id }: Props) {
   const router = useRouter();
-  const [form, setForm] = useState<Partial<CV>>({
+  const [form, setForm] = useState<Partial<any>>({
```

Comment:
I'm not opposed to kitchen sink states, but in React, usually, you would create an Education component that holds the `education` state:

```
    education: {
      university_name: "",
      start_year: "",
      end_year: "",
      degree: "",
    },
```

This way, you split the big form into smaller forms and have more granular states.

Reply:
Wouldn't it be more complicated to split the state here ? Then we would have to pass the state across components and merge it when we submit the form.

I created React components for separate sections on the form, but I did not split the state.